### ISSUE TITLE
And what about RIP asp.net webforms? 

### ISSUE THREAD

**Issue by alexsandro-xpt** [ThatRendle/ripgac#1]
It's become true? :D

**Comment by ThatRendle** [comment]
Ha, no. WebForms has a valid place in the world, it's still a pretty good way to build complicated custom Line of Business applications. But I do like the separation that vNext places between it and the more Web Site + API focused space.

**Comment by alexsandro-xpt** [comment]
I understand, but probably WebForms receive some little updates and become a legacy platform in the next years, did you agree?

**Comment by ThatRendle** [comment]
As I understand it, Microsoft are continuing to put significant effort into WebForms, at least for the immediate future. There are too many customers with too much invested in it to abandon it at this stage. But it will stay tied to the full .NET & Windows stack. The new MVC is aimed at a different market, one where portability, performance and OSS are key.

**Comment by alexsandro-xpt** [comment]
Yes! and today, there a lot of big customers working with VB6 yet. Some technologies never die.
